main-service/src/main/java/ru/practicum/users/dto/UserDto.java

```diff
@@ -0,0 +1,26 @@
+package ru.practicum.users.dto;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+
+@Setter
+@Getter
+@Builder
+public class UserDto {
+    private Long id;
+    @Size(min = 2, max = 250, message = "Имя должен иметь от 2 до 254 символов")
+    @NotBlank(message = "Имя не может состоять из пробелов")
+    private String name;
+    @Email(message = "Электронная почта заполнена неправильно")
+    @Size(min = 6, max = 254, message = "Электронная почта должна иметь от 2 до 254 символов")
```

Comment:
```suggestion
    @Size(min = 6, max = 254, message = "Электронная почта должна иметь от 6 до 254 символов")
```
значение в тексте и в коде отличается